src/utils/layout-helpers.ts

```diff
@@ -482,7 +495,13 @@ export function builtinModifiers(): CompletionItem[] {
   ];
 }
 
-export function listComponents(_root: string): CompletionItem[] {
+function hasNamespaceSupport(root: string) {
```

Comment:
could we add caching to this function, for 10 minutes or such, don't think this value may change very often